SubmittedAssignments/Sanjeevanee_balure_03_/index.html

```
@@ -0,0 +1,790 @@
+      <div class="card">
+        <div class="profile">
+            <img src="…?w=324&amp;h=192&amp;c=8&amp;rs=1&amp;qlt=90&amp;o=6&amp;dpr=1.3&amp;pid=3.1&amp;rm=2" alt="Corporate Events" />
+        <h3>Corporate Events</h3>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis.</p>
+        <span>Up to 100 guests</span>
+      </div>
+
+      <!-- Card 3 -->
+      <div class="card">
+        <div class="profile">
+		 <img src="https://th.bing.com/th/id/R.2605a9755405b2a3be476e33f2298d96?rik=1wj8XdbRlz7pEg&riu=http%3a%2f%2fwww.casalomaballroom.com%2fwp-content%2fuploads%2f2018%2f03%2fprivate-event2.jpg&ehk=SwdO1fQJjQZluBTnhGuyG57cx5NtB9ZjnCEgoePjMU0%3d&risl=&pid=ImgRaw&r=0"alt="Private Gatherings" /> 
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)